Tests/Error/ErrorHandlerTest.php

```diff
@@ -220,6 +230,11 @@ public function parentExceptionMappingDataProvider()
                 false,
                 ChildOfInvalidArgumentException::class,
             ],
+            'with $mapExceptionsToParent and no classes' => [
```

Comment:
@hgraca this covers your `return null` in the parent resolvers: https://github.com/overblog/GraphQLBundle/commit/44bea529ca7d8032f17e0652c83b03ae7b12f031#diff-7a5f7164895da8d47513f7ee5157841dR221

Reply:
Only now I had time to check it properly, good work :100: